public/css/style.css

```diff
@@ -0,0 +1,139 @@
+body,
+html {
+  background: #000;
+  position: relative;
+  margin: 0;
+  width: 100%;
+  height: 100%;
+}
+#mainC {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  -webkit-transform: translate(-50%, -50%);
+          transform: translate(-50%, -50%);
+  color: #fff;
+  font-family: sans-serif;
+  max-width: 340px;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -o-user-select: none;
+  -ms-user-select: none;
+      user-select: none;
+}
+#mainC .message {
+  font-size: 16px;
+  text-align: center;
+}
+#mainC .message h1 {
+  margin: 0;
+  padding: 0;
+  font-size: 11em;
+  -webkit-transform: skewY(-5deg);
+          transform: skewY(-5deg);
+  -webkit-transition: 0.4s ease-in-out all;
+  transition: 0.4s ease-in-out all;
+}
+#mainC .message h1:hover {
+  text-shadow: 20px 20px 0 rgba(175, 211, 61, 0.1);
+}
+@media (max-width: 300px) {
+  #mainC .message h1 {
+    font-size: 50vw;
+  }
+}
+#mainC .message h3 {
+  color: #afd33d;
+  font-size: 0.9em;
+  font-weight: lighter;
+  line-height: 1;
+}
+@media (max-width: 300px) {
+  #mainC .message h3 {
+    font-size: 5vw;
+  }
+}
+#mainC .footer {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -ms-flex-wrap: wrap;
+      flex-wrap: wrap;
```

Comment:
should we leave spaces as these?

Reply:
These are for clarification @abishekvashok :+1:

Reply:
@0x48piraj pls follow the coding style as far as possible. See the code in the repo.